doc/manual/noteworthy-differences.rst

```diff
@@ -251,3 +251,94 @@ Noteworthy differences from Python
+  64-bits, ``Float64``, and don't get promoted to the ``BigFloat`` type.  This is closer in
+  behavior to C/C++.
+  Octal (prefixed with ``0o``) and binary (prefixed with ``0b``) literals are also treated as unsigned
+  (with the same current exception that they become signed ``BigInt`` type if they can't fit in a ``UInt128``).
+- String literals can be delimited with either ``"``  or ``"""``, ``"""`` delimited literals can contain ``"``
+  characters without quoting it like ``"\\\""``
+  String literals can have values of other variables or expressions interpolated into them,
+  indicated by ``$variablename`` or ``$(expression)``, which evaluates the variable name or the expression in the context of the function.
+- ``//`` indicates a ``Rational`` number, and not a single-line comment (which is # in Julia)
+- ``#=`` indicates the start of a multiline comment, and ``=#`` ends it.
+- Functions in Julia return values from their last expression(s) or the ``return``
+  keyword.  Multiple values can be returned from functions and assigned as tuples, e.g.
+  ``(a, b) = myfunction()`` or ``a, b = myfunction()``, instead of having to pass pointers
+  to values as one would have to do in C/C++ (i.e. ``a = myfunction(&b)``.
+- Julia discourages the used of semicolons to end statements. The results of
+  statements are not automatically printed (except at the interactive prompt, i.e. the REPL),
+  and lines of code do not need to end with semicolons. :func:`println` or
+  :func:`@printf` can be used to print specific output.
+  In the REPL, ``;`` can be used to suppress output.
+  ``;`` also has a different meaning within ``[ ]``, something to watch out for.
+  ``;`` can be used to separate 'statements' on a line, but are not strictly necessary in many cases,
+  and are more an aid to readability.
+- In Julia, the operator :obj:`$` performs the bitwise XOR operation, i.e. :obj:`^`
+  in C/C++.  Also, the bitwise operators do not have the same precedence as C/++,
+  so parenthesis may be required.
+- Julia's :obj:`^` is exponentiation (pow), not bitwise XOR as in C/C++ (use :obj:`$` in Julia)
+- Julia has two right-shift operators, ``>>`` and ``>>>``.  ``>>>`` performs an arithmetic shift,
+  ``>>`` always performs a logical shift, unlike C/C++,
+  where the meaning of ``>>`` depends on the type of the value being shifted.
+- Julia's ``->`` creates an anonymous function, it does not access a member via a pointer.
+- Julia does not require parentheses when writing ``if`` statements or
+  ``for``/``while`` loops: use ``for i in [1, 2, 3]`` instead of
+  ``for (i in c(1, 2, 3))`` and ``if i == 1`` instead of ``if (i == 1)``.
+- Julia does not treat the numbers ``0`` and ``1`` as Booleans.
+  You cannot write ``if (1)`` in Julia, because ``if`` statements accept only
+  booleans. Instead, you can write ``if true``, ``if Bool(1)``, or ``if 1==1``.
+- In Julia, ``return`` does not require parentheses.
```

Comment:
Ah, again, overcopying on my part for the sections that seemed relevant from other languages... I'll fix.